src/views/ListOfBooks.jsx

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import "../styles/ListOfBooks.css";
+
+const ListOfBooks = () => {
+  const [books, setBooks] = useState(null);
+  const getListOfBooks = async () => {
+    let booksList;
+    await axios.post(
```

Comment:
Could you write this without using promises? Mixing async/await and promises is somewhat dangerous, and may force you to duplicate code.